_posts/2019-01-02-Deviceid-Reliability.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Keep the quality of your data high
+subtitle: All that glitters is not gold
+category: data
+tags: [adtech, howto]
+author: Alessio Borgheresi
+header-img: "images/Deviceid-Reliability/data_quality.jpg"
+---
+
+This post describes an example of the quality selection we perform at Adello, underlining the benefits we gain from it.
```

Comment:
I suggest to give it a bit more context: At Adello we are processing a lot of data. In a perfect world, we could just take that data and use it for our own and the benefit if our customers. But unfortunately, that data more often than not is not correct or incomplete...